tests/e2e/cucumber/scroll.spec.ts

```diff
@@ -1,16 +1,29 @@
 import { Then, When } from 'cypress-cucumber-preprocessor/steps';
 
 When('scrolling down to result {string}', (resultId: string) => {
-  cy.get(`[data-scroll=${resultId}]`)
-    .eq(0)
-    .scrollIntoView({ offset: { top: -5, left: 0 } })
-    .should('be.visible');
+  cy.get(`[data-scroll=${resultId}]`).eq(0).scrollIntoView();
 });
 
 Then('url is updated with result {string}', (resultId: string) => {
   cy.url().should('contain', `scroll=${resultId}`);
 });
 
+When('the page is reloaded', () => {
```

Comment:
Should this `When` be placed in the `common-steps` file?

Reply:
If it is only used here I don't see a reason to move it to the `common-steps` file.

Reply:
Although the action is generalist it's only being applied in this test and moving it could reduce step findability (translation: if we move it to `common-steps` it might take more time to find it than where it is now)